src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+goblin:
+  compendium-entry:
+    days-to-hard-delete: 30
+
+    cron:
+      cleanup-deleted-compendium-entries: "0 0 0 * * *"
```

Comment:
You can write comment explaining when exactly will this cron run, so that other developers don't have to decipher it.